app/web/src/components/EncodedUrl/ExternalScript.tsx

```diff
@@ -0,0 +1,126 @@
+import { useState } from 'react'
+import { useIdeContext, ideTypeNameMap } from 'src/helpers/hooks/useIdeContext'
+import OutBound from 'src/components/OutBound/OutBound'
+import { prepareEncodedUrl, makeExternalUrl } from './helpers'
+import { copyTextToClipboard } from 'src/helpers/clipboard'
+import { useRender } from 'src/components/IdeWrapper/useRender'
+import { toast } from '@redwoodjs/web/toast'
+
+const ExternalScript = () => {
```

Comment:
![image](https://user-images.githubusercontent.com/29681384/121826427-e7eea880-ccfa-11eb-9e5b-c82fd9605d87.png)
 and
 
![image](https://user-images.githubusercontent.com/29681384/121826437-f2a93d80-ccfa-11eb-9b3d-b5fa0e0968d3.png)